slick.grid.js

```diff
@@ -3185,6 +3185,11 @@ if (typeof Slick === "undefined") {
         if (i <= options.frozenColumn) {
           continue;
         }
+        
+        // Ignore alwaysRenderedColumns
+        if (columns[i].alwaysRenderColumn){
```

Comment:
please also make sure the `columns[i]` exist before calling it's property. A bullet proof check would be 
`if (Array.isArray(columns) && columns[i] && columns[i].alwaysRenderColumn) {`

Reply:
Added this and checked my page still works